qiskit/circuit/controlflow/__init__.py

```diff
@@ -24,4 +24,6 @@
 from .switch_case import SwitchCaseOp, CASE_DEFAULT
 
 
-CONTROL_FLOW_OP_NAMES = frozenset(("for_loop", "while_loop", "if_else", "switch_case"))
+CONTROL_FLOW_OP_NAMES = frozenset(
+    ("for_loop", "while_loop", "if_else", "switch_case", "continue_loop", "break_loop")
+)
```

Comment:
Fwiw, `ContinueLoopOp` and `BreakLoopOp` aren't subclasses of `ControlFlowOp`, which is what this was for checking, and they can't exist other than as the child of one of the others.  `ControlFlowOp` is about defining a small interface for instructions that contain other `QuantumCircuit`s as basic blocks.  These do impact the flow of control, but from the `QuantumCircuit` tree representation, that's often not actually particularly important (though I guess we wouldn't know - nothing supports them).

This probably doesn't hurt to add, but if anything's using this as a check and _not_ doing `isinstance(op, ControlFlowOp)` in addition, then it might cause breakage.